frontends/web/src/routes/settings/settings.tsx

```diff
@@ -362,6 +371,19 @@ class Settings extends Component<Props, State> {
                                                                 checked={config.backend.splitAccounts}
                                                                 onChange={this.handleToggleAccount} />
                                                         </div>
+                                                        <div className={style.currency}>
```

Comment:
Funny that frontend uses a style name `currency` for expert settings.

Reply:
It is funny, isn't it :joy: